index.js

```diff
@@ -172,7 +172,6 @@ function globStream(globs, opt) {
   var ourOpt = Object.assign(
     {},
     {
-      highWaterMark: 16,
```

Comment:
This was removed so we're using the defaults in streamx. I'm thinking we're not documenting changing the `highWaterMark` since users should not change it. We only expose it for testing.